src/docs/asciidoc/web/webflux.adoc

```diff
@@ -2279,6 +2290,9 @@ example uses a `BindingResult` argument`:
 	}
 ----
 
+And throws following exception :
+
+`java.lang.IllegalStateException: An Errors/BindingResult argument is expected immediately after the @ModelAttribute argument to which it applies. For @RequestBody and @RequestPart arguments, please declare them with a reactive type wrapper and use its onError operators to handle WebExchangeBindException`
 
```

Comment:
This looks wrong to me. The sample above shows how use BindingResult correctly. It shouldn't cause an ISE.

Reply:
Did you tried this pice of code?

Reply:
I wasted couple of hours for make @RequestBody and BindingResult  run in @RestController with this part from documentation. Not successful